tests/functional/u/unnecessary/unnecessary_ellipsis.py

```diff
@@ -9,10 +9,10 @@
     A = 2
 except ValueError:
     A = 24
-    ... # [unnecessary-ellipsis]
+    ...
```

Comment:
Hm, don't we think these are unnecessary?

Reply:
True. I simplified the checker to an extreme to prefer false negatives over false positives in order to make things as safe as possible to users. There will likely be other scenarios we can emit it also. We can allow this case in the current MR or in another one.

Reply:
Would `node.parent.parent.doc_node or len(node.parent.parent.body) > 1 work? Or something like that? Every `FunctionDef` or `ClassDef` that has a docstring or a body that has more than 1 node? That also reflects the description of the message a little better.

Reply:
That works well with the tests; lets try this. Thanks @DanielNoord.